### DISCUSSION TITLE
Does the Adobe Air Sdk can output gcf format which used by Steam？

### DISCUSSION THREAD

**Discussion by pengyangjin** [airsdk/Adobe-Runtime-Support#3153] [Q&A]
Does the adobe air sdk can output gcf format which used by Steam？
I want to publish games which is developed by Adobe AIR SDK on Steam.
Has anyone tried it? Is there any way?

**Comment by FliplineStudios** [comment]
You can just upload regular Windows and Mac builds to Steam to publish, I think they got rid of GCF format quite a few years ago unless there's still a special use-case for it these days.

**Comment by pengyangjin** [comment]
Thank you for your reply.




---Original---
From: ***@***.***&gt;
Date: Mon, Apr 1, 2024 22:05 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [airsdk/Adobe-Runtime-Support] Does the Adobe Air Sdk can output gcf format which used by Steam？ (Issue #3152)




 
You can just upload regular Windows and Mac builds to Steam to publish, I think they got rid of GCF format quite a few years ago unless there's still a special use-case for it these days.
